### DISCUSSION TITLE
Y-axis display format / Coordinate styling issue

### DISCUSSION THREAD

**Discussion by kaisiant** [reactivemarkets/react-financial-charts#477] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26655111/107196288-6970ba00-6a2d-11eb-861f-8aef38c82a09.png)

Hi, I am looking for help.

1) may I know how to set the formatter on the y-axis, I want to make it to 2 decimals points.
2) the coordinate width is overflowed, I tried fitToText but want to have some padding on the right.

**Comment by markmcdowell** [comment, marked as answer]
Hi, 
1. the `tickFormat` property sets the axis format. See the story as an example https://github.com/reactivemarkets/react-financial-charts/blob/master/packages/stories/src/features/StockChart.tsx#L106
2. Not sure what you mean here...?

Thanks!

**Reply by kaisiant** [reply]
1. Thanks
2. As you can see, the padding-left and right of the coordinate Y's container is not equal. What I am asking is, whether there is a way to solve it.

**Reply by markmcdowell** [reply]
You can set the `rectWidth` to change the indicator background.

**Reply by kaisiant** [reply]
If the `fitToText` is set, the rectWidth will not work.
Btw is the `ChartCanvas` margin issue, I have tuned it.
